zerver/openapi/zulip.yaml

```diff
@@ -12014,6 +12014,147 @@ paths:
                         description: |
                           An example JSON error response for when the user doesn't have permission
                           to subscribe other users to streams and `stream_ids` is not empty.
+  /invites/multiuse:
+    post:
+      operationId: create-invite-link
+      summary: Create reusable invitation link
+      tags: ["invites"]
+      description: |
+        Create a [reusable invitation link](/help/invite-new-users#create-a-reusable-invitation-link)
+        which can be used to invite new users to the organization.
+
+        **Changes**: In Zulip 8.0 (feature level 209), added support for non-admin
+        users to use this endpoint. Previously, it was restricted to administrators
```

Comment:
This note would benefit from linking to https://zulip.com/help/restrict-account-creation#change-who-can-send-invitations, maybe changing "non-admin users" to 

> non-admin users [with permission](https://zulip.com/help/restrict-account-creation#change-who-can-send-invitations) to use this endpoint